test/parallel/test-http-server-capture-rejections.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Should probably have a test for before headersSent and one for after.

Reply:
Done.